writerService.go

```diff
@@ -1,21 +1,79 @@
 package main
 
 import (
-	"fmt"
-	"io"
+	"github.com/influxdata/influxdb1-client/v2"
+	"log"
+	"strconv"
+	"time"
 )
 
-type MeasurementsWriterService struct{}
+type MeasurementsWriterService struct {
+	db           string
+	writerClient client.Client
+}
+
+func NewMeasurementsWriterService(dbAddress string) *MeasurementsWriterService {
+	clt, err := client.NewHTTPClient(client.HTTPConfig{
+		Addr: dbAddress,
+	})
+	if err != nil {
+		log.Panic(err)
+	}
+	return &MeasurementsWriterService{
+		db:           "mydb",
+		writerClient: clt,
+	}
+}
+
+func (mws *MeasurementsWriterService) dbWrite(batchPoints client.BatchPoints, measurement Measurement) {
+	point, err := client.NewPoint(
+		"deviceValues",
+		map[string]string{"deviceId": strconv.Itoa(measurement.Id)},
+		map[string]interface{}{"value": measurement.Value},
+		time.Now())
+	if err != nil {
+		log.Println(err)
```

Comment:
We won't know which record caused issues. My version would probably be:

`log.Printf("Could not save %+v: %s", measurement, err.Error())`